examples/logical_interconnect.pp

```diff
@@ -133,6 +133,15 @@
   }
 }
 
+oneview_logical_interconnect{'Bulk Inconsistency Validation':
+  ensure => 'bulk_inconsistency_validation_check',
```

Comment:
This should be bulk_inconsistency_validate

Reply:
fixed.